src/pybind/mgr/orchestrator/module.py

```diff
@@ -760,7 +760,7 @@ def _apply_mgr(self, placement=None):
 
     @_cli_write_command(
         'orch apply mon',
-        'name=placement,type=CephString,n=N,req=false',
+        'name=placement,type=CephString,req=false',
```

Comment:
this breaks the test of `tasks.mgr.test_orchestrator_cli.TestOrchestratorCli`:

```
2020-03-04T09:58:53.303 INFO:teuthology.orchestra.run.smithi203:> sudo adjust-ulimits ceph-coverage /home/ubuntu/cephtest/archive/coverage timeout 120 ceph --cluster ceph --log-early log 'Ended test tasks.mgr.test_orchestrator_cli.TestOrchestratorCli.test_mon_update'
2020-03-04T09:58:54.465 INFO:tasks.cephfs_test_runner:
2020-03-04T09:58:54.466 INFO:tasks.cephfs_test_runner:======================================================================
2020-03-04T09:58:54.466 INFO:tasks.cephfs_test_runner:ERROR: test_mon_update (tasks.mgr.test_orchestrator_cli.TestOrchestratorCli)
2020-03-04T09:58:54.466 INFO:tasks.cephfs_test_runner:----------------------------------------------------------------------
2020-03-04T09:58:54.466 INFO:tasks.cephfs_test_runner:Traceback (most recent call last):
2020-03-04T09:58:54.467 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2020-03-04-1337/qa/tasks/mgr/test_orchestrator_cli.py", line 160, in test_mon_update
2020-03-04T09:58:54.467 INFO:tasks.cephfs_test_runner:    self._orch_cmd("apply", "mon", "3", "host1:1.2.3.0/24", "host2:1.2.3.0/24", "host3:10.0.0.0/8")
2020-03-04T09:58:54.467 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2020-03-04-1337/qa/tasks/mgr/test_orchestrator_cli.py", line 21, in _orch_cmd
2020-03-04T09:58:54.467 INFO:tasks.cephfs_test_runner:    return self._cmd("orch", *args)
2020-03-04T09:58:54.467 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2020-03-04-1337/qa/tasks/mgr/test_orchestrator_cli.py", line 18, in _cmd
2020-03-04T09:58:54.467 INFO:tasks.cephfs_test_runner:    return self.mgr_cluster.mon_manager.raw_cluster_cmd(module, *args)
2020-03-04T09:58:54.467 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2020-03-04-1337/qa/tasks/ceph_manager.py", line 1359, in raw_cluster_cmd
2020-03-04T09:58:54.468 INFO:tasks.cephfs_test_runner:    stdout=BytesIO(),
2020-03-04T09:58:54.468 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_git_teuthology_master/teuthology/orchestra/remote.py", line 198, in run
2020-03-04T09:58:54.468 INFO:tasks.cephfs_test_runner:    r = self._runner(client=self.ssh, name=self.shortname, **kwargs)
2020-03-04T09:58:54.468 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_git_teuthology_master/teuthology/orchestra/run.py", line 433, in run
2020-03-04T09:58:54.468 INFO:tasks.cephfs_test_runner:    r.wait()
2020-03-04T09:58:54.468 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_git_teuthology_master/teuthology/orchestra/run.py", line 158, in wait
2020-03-04T09:58:54.468 INFO:tasks.cephfs_test_runner:    self._raise_for_status()
2020-03-04T09:58:54.468 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_git_teuthology_master/teuthology/orchestra/run.py", line 180, in _raise_for_status
2020-03-04T09:58:54.469 INFO:tasks.cephfs_test_runner:    node=self.hostname, label=self.label
2020-03-04T09:58:54.469 INFO:tasks.cephfs_test_runner:CommandFailedError: Command failed on smithi203 with status 22: 'sudo adjust-ulimits ceph-coverage /home/ubuntu/cephtest/archive/coverage timeout 120 ceph --cluster ceph --log-early orch apply mon 3 host1:1.2.3.0/24 host2:1.2.3.0/24 host3:10.0.0.0/8'
```

see `/a//kchai-2020-03-04_09:24:42-rados-wip-kefu-testing-2020-03-04-1337-distro-basic-smithi/4824403`

Reply:
fixed